src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,119 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int defaultSize = 10;
```

Comment:
The constant `defaultSize` should be named using uppercase letters with underscores, e.g., `DEFAULT_SIZE`, to follow Java naming conventions for constants.